pytorch_lightning/plugins/training_type/ddp.py

```diff
@@ -127,6 +128,7 @@ def __init__(
         self._model_averaging_period = model_averaging_period
         self._pids: Optional[List[int]] = None
         self._sync_dir: Optional[str] = None
+        self._init_process_group_kwargs = get_init_process_group_kwargs(kwargs)
```

Comment:
is it better to make `init_process_group_kwargs` an explicit argument to the plugin constructor?